### ISSUE TITLE
Redis does not start

### ISSUE THREAD

**Issue by challs** [MISP/x_old_misp_docker#72]
Redis does not start since 7c10e6801732192f488446ccd2b8e85f4deadb78. On the docker-compose console I see:

> misp_web | 2020-01-10 08:48:56,927 INFO exited: redis-server (exit status 1; not expected)
> misp_web | 2020-01-10 08:48:57,928 INFO gave up: redis-server entered FATAL state, too many start retries too quickly

In `/var/log/redis/redis-server.log` I see
```
231:M 10 Jan 09:50:17.552 # Creating Server TCP listening socket ::1:6379: bind: Cannot assign requested address
```
At first I thought it must be a local networking problem, but it went away again when I rolled back to the previous commit.

**Comment by fatalglitch** [comment]
The bind line in the /etc/redis/redis.conf includes ::1 for IPV6, which doesn't look to be supported in the docker container. The fix is to create another sed line in the web/Dockerfile to fix the redis config file

**Comment by fatalglitch** [comment]
After line 113 in web/Dockerfile, add the following and then run docker-compose -f <your compose file> build

RUN sed -i 's/^\(bind\s*\)127.0.0.1 ::1\s*$/\1127.0.0.1/g' /etc/redis/redis.conf

**Comment by Braincoke** [comment]
Thanks for the help. I think you forgot to escape the parentheses around `bind` in your answer. The line to add would be :

`RUN sed -i 's/^\(bind\s*\)127.0.0.1 ::1\s*$/\1127.0.0.1/g' /etc/redis/redis.conf`

I can confirm that adding this line solves the issue.

**Comment by fatalglitch** [comment]
Looks like github stripped the escape's, thanks for updating. This needs to be applied to the web/Dockerfile if someone can submit a PR for it

**Comment by challs** [comment]
Fixed by #75
